### DISCUSSION TITLE
Regarding Application

### DISCUSSION THREAD

**Discussion by CoderShubham2000** [github/india#584] [Students]
Can students who were already a part of the previous externship apply for this cohort?

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any `Official GitHub Externship Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Link to join: https://discord.gg/ceMXzhfaka

Thank you 🙏

**Comment by harshita130602** [comment]
The discord is showing that the invite is invalid. Pls help
